src/GoogleAdWordsEventForwarder.js

```diff
@@ -53,17 +53,13 @@
 
                 try {
                     if (window.gtag && forwarderSettings.enableGtag == 'True') {
-                        if (event.CustomFlags && 
-                            Object.keys(event.CustomFlags).length &&
-                            event.CustomFlags[ENHANCED_CONVERSION_DATA]
-                        ) {
+                        if (hasEnhancedConversionData(event.CustomFlags)) {
```

Comment:
Optimization: Line 56 can go below line 57.  If they don't have the setting checked, then we don't have to run any code to see if the `ECData` custom flag exists at all.

Reply:
Agreed. I assumed there was a legacy reason for checking it this way.